app/src/main/java/com/example/playlistmaker/ui/root/MainActivity.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.playlistmaker.ui.root
+
+import android.os.Bundle
+import android.view.View.GONE
+import android.view.View.VISIBLE
+import androidx.appcompat.app.AppCompatActivity
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.ui.setupWithNavController
+import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.ActivityMainBinding
+import com.example.playlistmaker.ui.player.PlayerFragment
+
+class MainActivity : AppCompatActivity() {
+    private val binding by lazy { ActivityMainBinding.inflate(layoutInflater) }
```

Comment:
У `lazy` в данном случае убрать LazyThreadSafetyMode, чтобы отключить синхронизацию доступа к лямбде, т.к. тут только 1 поток получает доступ к этому полю:

```suggestion
    private val binding by lazy(mode = LazyThreadSafetyMode.NONE) { ActivityMainBinding.inflate(layoutInflater) }
```